sync/class.jetpack-sync-actions.php

```diff
@@ -26,6 +26,8 @@ static function init() {
 					defined( 'DOING_AJAX' ) && DOING_AJAX
 				||
 					defined( 'PHPUNIT_JETPACK_TESTSUITE' )
+				||
+				    is_admin()
```

Comment:
Indentation seems off compared to the other checks. Other than that, this seems good 👍